site/app/views/NavigationView.php

```diff
@@ -73,7 +73,8 @@ class NavigationView extends AbstractView {
 
     const DATE_FORMAT = "m/d/Y @ h:i A";
 
-    public function showGradeables($sections_to_list, $graded_gradeables, array $submit_everyone, $gradeable_ids_and_titles, $teams_viewed_times, $gradables_teams) {
+    public function showGradeables($sections_to_list, $graded_gradeables, array $submit_everyone,
+                                   $gradeable_ids_and_titles, $gradables_teams, $teams_viewed_times) {
```

Comment:
Types are usually a good thing to declare if they aren't mixed. But I guess it's good enough